_docs/layout/opener-images.md

```diff
@@ -7,14 +7,21 @@ order: 3
 
 # Page-opener images
 
-You can add a single, big opening image to each page. This image is set in the page's top-of-page YAML. E.g. the landing page for the template has:
+You can add a single, big opening image to each page. This image is set in the page's top-of-page YAML. You can define:
+
+- the image to use as `opener-image`
+- alt text for the image as `opener-image-alt-text`
+- which area of the image to focus on, when the screen size and shape crops the image, as `opener-image-position`. This follows the properties for [CSS object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position).
+
+For example, the landing page for the template has:
 
 ```YAML
-opener-image: banner-image.jpg
+opener-image: "banner-image.jpg"
 opener-image-alt-text: "An illustration of an enormous book. It is open, and its pages are each filled with an image of a starry sky. Beside the book, a man stands and looks at the pages. The book is bigger than he is."
+opener-image-position: 0 60%;
```

Comment:
I think this might need quotation marks around the value, as they are present in the markdown file.

Reply:
Thanks! The semi-colon is a mistake, oops. While the quote marks are technically not necessary, it's useful to be in the habit of including them to avoid breaking YAML syntax, e.g. with colons in values. I do like to include the quotes in docs. Fixed.